hwilib/devices/bitbox02.py

```diff
@@ -732,7 +746,19 @@ def script_config_from_utxo(
         for output_index, psbt_out in builtins.enumerate(psbt.outputs):
             tx_out = psbt_out.get_txout()
 
-            _, keypath = find_our_key(psbt_out.hd_keypaths)
```

Comment:
on line 721 above, prevtx can be None, so should be:

```
"prev_tx": None if prevtx is None else { .... }
```

Reply:
Thx, updated it!